pkg/elle/core/history.go

```diff
@@ -96,16 +116,47 @@ func (m Mop) IsEqual(n Mop) bool {
 func (m Mop) String() string {
 	switch m.T {
 	case MopTypeRead:
-		key := m.M["key"].(string)
-		value := m.M["value"]
-		if value == nil {
-			return fmt.Sprintf("[:r %s nil]", key)
+		key, ok := m.M["key"]
+		if ok {
+			value := m.M["value"]
+			if value == nil {
```

Comment:
(Maybe just `fmt.Sprintf("[:r %s %v]", key, value)` is ok)

Reply:
It'll be `[:r x <nil>]` if so, but I want `[:r x nil]`